src/components/attendee/WebcamGrid/WebcamGrid.tsx

```diff
@@ -24,26 +24,15 @@ export const WebcamGrid: React.FC<TableGridProps> = ({ space }) => {
     localParticipant,
     remoteParticipants,
     disconnect,
-    startVideo,
-    startAudio,
-    stopVideo,
-    stopAudio,
   } = useVideoComms();
 
   useEffect(() => {
-    // Only connect once the user ID is set (loaded)
-    if (userId) {
-      joinChannel({
-        userId,
-        channelId: `webcamgrid-${space.id}`,
-        enableAudio: false,
-        enableVideo: false,
-      });
-      return () => {
+    return () => {
+      if (hasJoined) {
         disconnect();
-      };
-    }
-  }, [disconnect, joinChannel, space.id, userId]);
```

Comment:
This change turns off the auto-joining behaviour which has been problematic. We can come back and do a more complex fix to this if the product team want us to.

Reply:
I propose ` if( !DISABLED_DUE_TO_AUTOJOIN_PROBLEMS && userId)` check before joining instead of outright deleting this code. The explanation can be put in short or long form in `disableSettings.ts` (the primary reason for that file existence)

Reply:
I'm not adding more flakes and more conditionals. If the product team want this it'll need fixing properly.